### ISSUE TITLE
Support for drilldown links in charts of `type = MARKDOWN`

### ISSUE THREAD

**Issue by akshaan** [vmware/terraform-provider-wavefront#7]
I'm attempting to create a chart of type `MARKDOWN` with a drilldown link pointing at another dashboard. Additionally I'd like to have variables transferred from the main dashboard to the linked dashboard when clicking out of the chart. This can be done in the UI using the editing flow for the chart (screenshot below):

<img width="1200" alt="Screen Shot 2020-08-24 at 1 56 47 PM" src="https://user-images.githubusercontent.com/3531294/91079533-27d08c00-e612-11ea-93c4-0e446d082345.png">

These attributes show up in the chart JSON under the `chartAttributes` dictionary as follows:
```
...
"chartSettings" = {...}
"chartAttributes": {
                "dashboardLayout": {
                  "x": 0,
                  "y": 0,
                  "w": 3,
                  "h": 5
                },
                "dashboardLinks": {
                  "*": {
                    "destination": "/dashboards/inference-server-predict-model-metrics",
                    "variables": {
                      "env": "env"
                    }
                  }
                }
              },
```

I don't see support for manipulating the `chartAttributes` section through terraform. Is this possible to do with the current version of the provider?

**Comment by keep94** [comment]
We released version 2.7.2 yesterday which supports chart attributes in dashboards.

**Comment by fishy** [comment]
@keep94 searching for "attribute" on the [official documentation page](https://registry.terraform.io/providers/vmware/wavefront/latest/docs/resources/dashboard) returned nothing. Can you give some examples on how to do this with the terraform provider?
